README.md

```diff
@@ -11,6 +9,9 @@ The project consists of three main parts:
 - Rest Gateway, that receive the lineage data from agent and stores it in the database
 - Web UI application that visualizes the stored data lineages
 
+![Spline diagram](https://user-images.githubusercontent.com/5530211/69061052-c4795400-0a18-11ea-9990-91836e2d648a.png)
+
```

Comment:
I think we should provide more information on how Spline is supposed to be used: 
- Spark Agent is declared as dependency in your Spark app. 
- Rest Gateway and Spline UI are war files that needs to run in servlet container like Tomcat

Reply:
For spark Agent, there is a part : Use spline in your application 

And for Rest Gateway and spline UI, there is all info  in "Start Spline Server" and "Start Spline UI" parts

Reply:
Ok, I guess the image should be enough for overview